KUBERNETES.md

```diff
@@ -122,6 +131,41 @@ sbt clean docker:publishLocal
 docker images
 ```
 
+##### Configure RBAC
```

Comment:
To reiterate my comment in https://github.com/lightbend/lightbend-orchestration-docs/pull/39, I'm worried that having this information duplicated in multiple places might cause it to diverge in the future, or at least adds extra maintenance burden.

Maybe we should have a single guide for setting up a Kubernetes cluster for RP apps, and link to that as a prerequisite from other places.